app/uk/gov/hmrc/nisp/views/statePension/StatePensionMqpView.scala.html

```diff
@@ -0,0 +1,264 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.nisp.controllers.auth.{AuthenticatedRequest, NispAuthedUser}
+@import uk.gov.hmrc.nisp.controllers.routes
+@import uk.gov.hmrc.nisp.models.StatePension
+@import uk.gov.hmrc.nisp.models.enums.MQPScenario._
+@import uk.gov.hmrc.nisp.views.Main
+@import uk.gov.hmrc.nisp.views.html.includes.newComponents._
+@import uk.gov.hmrc.nisp.views.html.includes.{NiLInks, statePensionAgeUnderConsideration}
+@import uk.gov.hmrc.nisp.views.html.statePension.StatePensionFooter
+@import uk.gov.hmrc.play.language.LanguageUtils
+
+@this(
+        main: Main,
+        statePensionAgeUnderConsideration: statePensionAgeUnderConsideration,
+        langUtils: LanguageUtils,
+        p: P,
+        h1: H1,
+        h2: H2,
+        h3: H3,
+        link: Link,
+        govukWarningText: GovukWarningText,
+        statePensionFooter: StatePensionFooter,
+        printLink: PrintLink,
+        span: Span,
+        niLinks: NiLInks
+)
+@(
+        statePension: StatePension,
+        numberOfGaps: Int,
+        numberOfPayableGaps: Int,
+        numberOfYearsMissing: Int,
+        customerAge: Int
+)(
+        implicit request: AuthenticatedRequest[_],
+        user: NispAuthedUser,
+        messages: Messages
+)
+
+@pensionDate = @{
+    langUtils.Dates.formatDate(statePension.pensionDate)
+}
+
+@defining(Some(user)) { _ =>
+    @main(
+        pageTitle = messages("nisp.main.title")
+    ) {
+        <p class="govuk-body" id="which-view">New MQP View</p>
```

Comment:
Are we leaving this in?

Reply:
Yeah these are for testing purposes prior to switch on, they will come out later on